src/main/scala/firrtl/Emitter.scala

```diff
@@ -232,6 +232,11 @@ case class VRandom(width: BigInt) extends Expression {
 
 object VerilogEmitter {
 
+  private val unaryOps: Set[PrimOp] = Set(Andr, Orr, Xorr, Neg, Not)
+
+  // To make uses more self-documenting
+  def isUnaryOp: PrimOp => Boolean = unaryOps
```

Comment:
make this private?

Reply:
Oh yeah, it was originally but I was messing with it. It's also supposed to be a `val`, thanks!